delphi-checks/src/test/java/au/com/integradev/delphi/checks/DigitGroupingCheckTest.java

```diff
@@ -36,6 +36,10 @@ class DigitGroupingCheckTest {
         "$ABC_DEF",
         "$ABC_DE_F",
         "%1_001_01",
+        "&1_001_01",
```

Comment:
nit: these look like a binary literal at first, can you change the non-leading 1s to 0?